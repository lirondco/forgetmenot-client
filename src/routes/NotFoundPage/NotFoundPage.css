.error_title {
    font-family: 'Patrick Hand SC', cursive;
    padding: 5%;
    margin: 5%;
}

.error_text {
    font-family: 'Mina', sans-serif;
    padding: 5%;
    margin: 5%;
}