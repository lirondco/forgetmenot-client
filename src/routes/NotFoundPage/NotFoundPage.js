import React, { Component } from 'react'
import { Section } from '../../components/Utils/Utils'
import './NotFoundPage.css'

export default class NotFoundPage extends Component {
  render() {
    return (
      <Section className='NotFoundPage'>
        <h2 className='error_title'>Four-Oh-Four Error - Uh oh!</h2>
        <p className='error_text'>I don't think this was supposed to happen. Please check your URL and try again</p>
      </Section>
    )
  }
}