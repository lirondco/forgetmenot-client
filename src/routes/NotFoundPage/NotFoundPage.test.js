import React from 'react';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import NotFoundPage from './NotFoundPage'

describe(`NotFoundPage component`, () => {
  it('renders the complete error message', () => {
    const wrapper = shallow(<NotFoundPage />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})