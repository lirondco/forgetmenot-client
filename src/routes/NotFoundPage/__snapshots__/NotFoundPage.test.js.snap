// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage component renders the complete error message 1`] = `
<Section
  className="NotFoundPage"
>
  <h2
    className="error_title"
  >
    Four-Oh-Four Error - Uh oh!
  </h2>
  <p
    className="error_text"
  >
    I don't think this was supposed to happen. Please check your URL and try again
  </p>
</Section>
`;
